Projects/DesignSystem/Sources/Component/MainTitleView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  MainTitleView.swift
+//  DesignSystem
+//
+//  Created by 김태호 on 2022/10/20.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+/// Main Title을 만들기 위한 UIView
+/// - title: 화면의 타이틀에 들어갈 문자열
+/// - subtitle: (옵셔널) 화면의 타이틀 밑에 들어갈 회색의 문자열
+///
+/// 높이는 최소 94 point로 정해져 있습니다.
+///
+/// top constraint와 너비만 지정해주면 됩니다
+public final class MainTitleView: UIView {
+    private let titleLabel = UILabel()
+    private let subtitleLabel = UILabel()
+    private let titleStackView = UIStackView()
+    
+    public init(title: String, subtitle: String? = "") {
```

Comment:
[P3]

UILabel처럼 나중에도 타이틀과 서브 타이틀을 수정할 수 있게 해주면 좋을 것 같습니다!
컴포넌트의 경우에는 디자인에 따라 유동성이 있어도 좋다고 생각해요

```suggestion
public final class MainTitleView: UIView {
    public var titleLabel = UILabel()
    public var subtitleLabel = UILabel()
    private let titleStackView = UIStackView()
    
    public init(title: String = "", subtitle: String? = "") {
```

Reply:
4eba8f5 변경하였습니다!

생성자의 경우 이 컴포넌트의 역할이 title을 만든는 것이기 때문에 title에는 기본값을 두지 않았습니다.
이 컴포넌트를 사용하는 개발자에게 컴포넌트의 역할과 컴포넌트의 동작인 title 입력을 해야 한다는 것을 명시하기 위함입니다!

Reply:
여전히 자유를 더 줘도 괜찮다고는 생각하지만,,
기본 컴포넌트와 다르게 레이아웃이 잡혀있는 상태이니 title 입력을 강제하는 게 낫다는 생각도 드네요
반영 감사합니다!